lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -6394,6 +6394,39 @@ void HierarchicalModuleNameOp::getAsmResultNames(
   setNameFn(getResult(), "hierarchicalmodulename");
 }
 
+//===----------------------------------------------------------------------===//
+// BindOp
+//===----------------------------------------------------------------------===//
+
+namespace {
+template <class Op>
+Op findInnerSym(StringAttr name, Block *body) {
+  for (auto &op : llvm::reverse(body->getOperations()))
+    if (auto target = dyn_cast<Op>(op))
+      if (auto innerSym = target.getInnerSym())
+        if (innerSym->getSymName() == name)
+          return target;
+  return {};
+}
```

Comment:
This is _very_ expensive.

Can this be changed to use `verifyInnerRefs` so that this is done efficiently?

Reply:
Thanks, I didn't know about this trait/verification method. The [sv.bind op](https://github.com/llvm/circt/blob/main/include/circt/Dialect/SV/SVStatements.td#L576) is not using verifyInnerRefs either. I'm not sure if this is intentional or an oversight.

Reply:
The SV bind op is ancient and doesn't represent modern conventions. The fact that it doesn't implement this may stem from the problem of it being rooted in a `builtin.module` which restricts our ability to do verification.

Reply:
Yep that seems to be the case, the firrtl CircuitOp is an InnerRefNamespace and the MLIR ModuleOp cannot be.